src/main/cpp/hw/DragonSparkFlex.cpp

```diff
@@ -198,25 +177,27 @@ void DragonSparkFlex::SetVoltage(units::volt_t output)
 
 bool DragonSparkFlex::IsMotorInverted() const
 {
+    return m_spark->GetInverted();
 }
 
 bool DragonSparkFlex::IsForwardLimitSwitchClosed()
 {
-    return false;
+    return m_spark->GetForwardLimitSwitch(m_forwardType).Get();
```

Comment:
This may have the same problem as GetEncoder() where GetForwardLimitSwitch() cannot be called more than once and we need to store the value.